quiche/http2/adapter/header_validator_test.cc

```diff
@@ -39,7 +39,7 @@ TEST(HeaderValidatorTest, ExceedsMaxSize) {
   EXPECT_EQ(HeaderValidator::HEADER_OK, status);
   status = v.ValidateSingleHeader(
       "name2",
-      "Antidisestablishmentariansism is supercalifragilisticexpialodocious.");
+      "Antidisestablishmentariansism is supercalifragilisticexpealidocious.");
```

Comment:
both tokens were present in this repository. There was no obvious explanation for why this incorrect spelling was present.